PySDM/backends/impl_numba/methods/seeding_methods.py

```diff
@@ -14,10 +14,18 @@ def _seeding(self):
         def body(  # pylint: disable=too-many-arguments
             idx,
             multiplicity,
+            cell_id,
+            cell_origin,
+            pos_cell,
+            volume,
             extensive_attributes,
             seeded_particle_index,
             seeded_particle_multiplicity,
+            seeded_particle_cell_id,
+            seeded_particle_cell_origin,
+            seeded_particle_pos_cell,
             seeded_particle_extensive_attributes,
+            seeded_particle_volume,
```

Comment:
shouldn't the volume be one of the extensive attributes?

Reply:
volume is being derived "on the fly" from mass, it is a derived attribute, and is thus not included in the base attribute set (base = not derived)